ayon_api/server_api.py

```diff
@@ -367,10 +367,10 @@ class ServerAPI(object):
         sender (Optional[str]): Sender of requests. Used in server logs and
             propagated into events.
         ssl_verify (Union[bool, str, None]): Verify SSL certificate
-            Looks for env variable value 'AYON_CA_FILE' by default. If not
+            Looks for env variable value ``AYON_CA_FILE`` by default. If not
```

Comment:
Why double ` `` `?

Reply:
that relates how it would be formated, similar to markdown. It will do `foo` instead of 'foo'